libs/filer/filer.go

```diff
@@ -10,14 +10,14 @@ import (
 type WriteMode int
 
 const (
-	OverwriteIfExists       WriteMode = iota
-	CreateParentDirectories           = iota << 1
+	OverwriteIfExists WriteMode = 1 << iota
+	CreateParentDirectories
```

Comment:
These changes were necessary to make the mocks work; turns out they were incorrect to begin with.